content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Control Topology Management Policies on a node
+reviewers:
+- lmdaly
+- nolancon
+- klueska
+- ConnorDoyle
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+An increasing number of systems leverage a combination of CPUs and hardware accelerators to support latency-critical execution and high-throughput parallel computation. These include workloads in fields such as telecommunications, scientific computing, machine learning, financial services and data analytics. Such hybrid systems comprise a high performance environment.
+
+In order to extract the best performance, optimizations related to CPU isolation and memory and device locality are required. However, in Kubernetes, these optimizations are handled by a disjoint set of components.
+
+_Topology Manager_ is a component in Kubelet that provides node level policies to enable these performance optimizations in an user abstract manner.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Topology Manager
+
+Currently, the CPU and Device Manager in Kubernetes make resource allocation decisions independently of each other.

```

Comment:
> Currently
“Previously”? Or perhaps “If you don't already use Topology Manager in your cluster, ”?

Reply:
Updated, let me know what you think!